subprojects/test-runner/device-provider/src/main/kotlin/com/avito/android/runner/devices/internal/Wait.kt

```diff
@@ -13,9 +13,14 @@ internal suspend fun <T> waitForCondition(
     successMessage: String? = null,
     maxAttempts: Int = WAIT_FOR_COMMAND_MAX_ATTEMPTS,
     frequencySeconds: Long = WAIT_FOR_COMMAND_FREQUENCY_SECONDS,
+    timeoutSec: Long = WAIT_FOR_COMMAND_TIMEOUT,
     condition: suspend () -> Result<T>
 ): Result<T> {
```

Comment:
can we reuse `com.avito.android.waiter.waitFor`  or at least move this function to `:common:waiter` ?

Reply:
I tried but they too different now, maybe next time